host/linux/time.c

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#include "openenclave/bits/time.h"
```

Comment:
move down into alphabetical order and make all OE headers be consistent (I'd prefer all "" not <>)